attributes/monitors.rb

```diff
@@ -61,12 +61,12 @@
   chk['details']['timeout'] = 60
   # Platformstack wants the alarm hash
   # Can override this in the wrapper/role/env
-  chk['alarm'] = {}
+  chk['alarm'] = false
```

Comment:
On platformstack `alarm` was the full alarm criteria, on rackspace_monitoring `alarm` is to enable/disable. I've not found a way to make it backward compatible with platformstack.